lib/material/internal/material_tabs.flow

```diff
@@ -145,27 +145,27 @@ MDynamicTabs2T(manager : MaterialManager, parent : MFocusGroup, m : MDynamicTabs
 					maxLines = extractStruct(tb.style, MMaxLines(2)).lines;
 					widthByContent = contains(tb.style, MWidthByContent());
 					centerTitle = !(titleAlignIcon && isSome(tabIcon));
-
 					availableW = make(0.);
 					fullWidth = make(0.);
 					iconW = make(0.);
 					isIconVisible = make(true);
 					dynArrayPush(iconsVisible, isIconVisible);
 
 					TSelect2(tb.text, fand(const(!isKeepTabColor), feq(m.selected, i)), \txt, selected ->
-						if (txt != "") {
-							color = if (selected) Some(indicatorColor) else itemsColor;
+						TShow(const(txt != ""), {
+							if (isSome(iconAlign)) TFillX() 
```

Comment:
Bug was introduced with the changed order of branches - text will disappear with MTabsIconAlign present.